### ISSUE TITLE
Accessibility: <html> element does not have a [lang] attribute

### ISSUE THREAD

**Issue by gabek** [owncast/owncast#2445]
## `<html>` element does not have a [lang] attribute

> If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly.

Re: #1826

**Comment by silksow** [comment]
pls assign
